runtime/test/test-helper.js

```diff
@@ -0,0 +1,135 @@
+/**
+ * @license
+ * Copyright (c) 2017 Google Inc. All rights reserved.
+ * This code may only be used under the BSD style license found at
+ * http://polymer.github.io/LICENSE.txt
+ * Code distributed by Google as part of this project is also
+ * subject to an additional IP rights grant found at
+ * http://polymer.github.io/PATENTS.txt
+ */
+'use strict';
+
+import {assert} from './chai-web.js';
+import Arc from '../arc.js';
+import Manifest from '../manifest.js';
+import Loader from '../loader.js';
+import Planner from '../planner.js';
+import MockSlotComposer from './mock-slot-composer.js';
+
+class TestHelper {
```

Comment:
I think you can just put `export default class TestHelper { ...` here, and remove the `export` at end of file, and that's idiomatic for files that only export one class defined therein.

Reply:
done, thanks!